### ISSUE TITLE
Unable to start VM - This computer doesn't have VT-X/AMD-v enabled

### ISSUE THREAD

**Issue by mrtkp9993** [kubernetes/minikube#4816]
I am trying to start ```minikube``` and got following error:

```
$ minikube start
😄  minikube v1.2.0 on linux (amd64)
💿  Downloading Minikube ISO ...
 129.33 MB / 129.33 MB [============================================] 100.00% 0s
🔥  Creating virtualbox VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...
E0719 10:04:35.259838    4579 start.go:559] StartHost: create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory

💣  Unable to start VM
❌  Error:         [VBOX_VTX_DISABLED] create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory
💡  Advice:        In some environments, this message is incorrect. Try 'minikube start --no-vtx-check'
⁉️   Related issues:
    ▪ https://github.com/kubernetes/minikube/issues/3900

😿  If the above advice does not help, please let us know: 
👉  https://github.com/kubernetes/minikube/issues/new
```

My system information:

```
$ screenfetch 

 ██████████████████  ████████     muratk@murat-pc
 ██████████████████  ████████     OS: Manjaro 18.0.4 Illyria
 ██████████████████  ████████     Kernel: x86_64 Linux 4.19.59-1-MANJARO
 ██████████████████  ████████     Uptime: 42m
 ████████            ████████     Packages: 1376
 ████████  ████████  ████████     Shell: bash 5.0.7
 ████████  ████████  ████████     Resolution: 1366x768
 ████████  ████████  ████████     DE: Xfce4
 ████████  ████████  ████████     WM: Xfwm4
 ████████  ████████  ████████     WM Theme: Adapta-Maia
 ████████  ████████  ████████     GTK Theme: Matcha-dark-azul [GTK2]
 ████████  ████████  ████████     Icon Theme: Papirus-Maia
 ████████  ████████  ████████     Font: Noto Sans 10
 ████████  ████████  ████████     CPU: AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx @ 4x 2.5GHz
                                  GPU: AMD RAVEN (DRM 3.27.0, 4.19.59-1-MANJARO, LLVM 8.0.0)
                                  RAM: 2294MiB / 7584MiB

```

Virtualization seems enabled:

```
$ lscpu 
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   43 bits physical, 48 bits virtual
CPU(s):                          4
On-line CPU(s) list:             0-3
Thread(s) per core:              2
Core(s) per socket:              2
Socket(s):                       1
NUMA node(s):                    1
Vendor ID:                       AuthenticAMD
CPU family:                      23
Model:                           17
Model name:                      AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx
Stepping:                        0
Frequency boost:                 enabled
CPU MHz:                         1374.256
CPU max MHz:                     2500,0000
CPU min MHz:                     1600,0000
BogoMIPS:                        4992.36
Virtualization:                  AMD-V
L1d cache:                       64 KiB
L1i cache:                       128 KiB
L2 cache:                        1 MiB
L3 cache:                        4 MiB
NUMA node0 CPU(s):               0-3
Vulnerability L1tf:              Not affected
Vulnerability Mds:               Not affected
Vulnerability Meltdown:          Not affected
Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl and seccomp
Vulnerability Spectre v1:        Mitigation; __user pointer sanitization
Vulnerability Spectre v2:        Mitigation; Full AMD retpoline, IBPB conditional, STIBP disabled, RSB filling
Flags:                           fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht s
                                 yscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid a
                                 perfmperf pni pclmulqdq monitor ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand la
                                 hf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw skinit wdt tce topoex
                                 t perfctr_core perfctr_nb bpext perfctr_llc mwaitx cpb hw_pstate sme ssbd sev ibpb vmmcall fsgsbase bm
                                 i1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xsaveopt xsavec xgetbv1 xsaves clzero irperf xsave
                                 erptr arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasid decodeassists pausefilter pfth
                                 reshold avic v_vmsave_vmload vgif overflow_recov succor smca
```

**Comment by afbjorklund** [comment]
This is an unfortunate regression, from moving to a libmachine without a fix for https://github.com/docker/machine/issues/4669

The workaround is to use `minikube start --no-vtx-check`.  The originial minikube issue is: https://github.com/kubernetes/minikube/issues/3706

**Comment by afbjorklund** [comment]
Moving forward, we could move to the latest libmachine `master` where the fix should be included now.

Since https://github.com/docker/machine/commit/2b369175bc5ae80ad9d98d069059af1a17350244

**Comment by marinetteman2** [comment]
Still occurring in 1.2.2 minikube

**Comment by Economax** [comment]
Still occuring in minikube v 1.17.1 on Microsoft Windows 10 Home 10.0.19041 build 19041 with AMD processor Ryzen 7 1700X and WSL 2 installed.

**Comment by afbjorklund** [comment]
> Still occuring in minikube v 1.17.1 on Microsoft Windows 10 Home 10.0.19041 build 19041 with AMD processor Ryzen 7 1700X and WSL 2 installed.

@Economax Can you open a new issue, if so ? It's possible that `cpuid` needs some update for newer AMD...

**Comment by lironmo** [comment]
PS C:\WINDOWS\system32> minikube start
* minikube v1.24.0 on Microsoft Windows 10 Home 10.0.19043 Build 19043
* Using the virtualbox driver based on existing profile
* Starting control plane node minikube in cluster minikube
* Creating virtualbox VM (CPUs=2, Memory=5000MB, Disk=20000MB) ...
! StartHost failed, but will try again: creating host: create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory
* Creating virtualbox VM (CPUs=2, Memory=5000MB, Disk=20000MB) ...
* Failed to start virtualbox VM. Running "minikube delete" may fix it: creating host: create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory

**Comment by gcsaba2** [comment]
PS C:\WINDOWS\system32> minikube start
* minikube v1.25.2 on Microsoft Windows 10 Pro 10.0.19044 Build 19044
* Using the virtualbox driver based on existing profile
* Starting control plane node minikube in cluster minikube
* Creating virtualbox VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
! StartHost failed, but will try again: creating host: create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory
* Creating virtualbox VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
* Failed to start virtualbox VM. Running "minikube delete" may fix it: creating host: create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory

X Exiting due to HOST_VIRT_UNAVAILABLE: Failed to start host: creating host: create: precreate: This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory
* Suggestion: Virtualization support is disabled on your computer. If you are running minikube within a VM, try '--driver=docker'. Otherwise, consult your systems BIOS manual for how to enable virtualization.
* Related issues:
  - https://github.com/kubernetes/minikube/issues/3900
  - https://github.com/kubernetes/minikube/issues/4730

**Comment by laksharmaa** [comment]
> This is an unfortunate regression, from moving to a libmachine without a fix for [docker/machine#4669](https://github.com/docker/machine/issues/4669)
> 
> The workaround is to use `minikube start --no-vtx-check`. The original minikube issue is: #3706

it helped...thank you
